tools/dfu-util/meta.yaml

```diff
@@ -19,7 +19,12 @@ build:
 
 requirements:
   build:
+    # On macOS `autoconf`, `automake`, `coreutils`
+    # and `libtool` need to be installed from brew.
```

Comment:
why not like in openocd?

Reply:
I've tried to make it like `openocd` but I couldn't make the build fully successful. Even with only `autoconf` and `automake` from Conda there was some mysterious failure.